src/Theme.jsx

```diff
@@ -42,12 +42,12 @@ export default function Theme({ theme, handleOnDelete, handleOnEdit }) {
             />
           </div>
           <div className="buttons">
-          <button className="delete-button" type="button" onClick={onDelete}>
-            Delete
-          </button>
-          <button type="button" onClick={onEdit}>
-            Edit
-          </button>
+            <button className="delete-button" type="button" onClick={onDelete}>
```

Comment:
typically, the internal handler functions that you write here in the component are called `handleXYZ`. `onDelete` or `onEdit` suggest that these are the props. This is only a convention, but other react devs would expect it like this.